libnetwork/networkdb/cluster.go

```diff
@@ -520,10 +520,7 @@ func (nDB *NetworkDB) bulkSyncTables() {
 	}
 	nDB.RUnlock()
 
-	for {
-		if len(networks) == 0 {
-			break
-		}
+	for len(networks) != 0 {
 
```

Comment:
If you're updating; this empty line can probably be removed (we don't enforce `gofumpt` yet, but it very likely would remove this line.